src/core/tsi/ssl_transport_security.cc

```diff
@@ -2212,7 +2212,7 @@ static void tsi_ssl_server_handshaker_factory_destroy(
   if (self->alpn_protocol_list != nullptr) gpr_free(self->alpn_protocol_list);
   self->key_logger.reset();
```

Comment:
Don't actually need these two lines anymore, since the dtors will now be called.